handlers/delete_lease.go

```diff
@@ -48,17 +46,17 @@ func DeleteLease(services k8s.ServiceGetterUpdater, k8sServiceName string, names
 		}
 
 		// clear all namespaces
-		namespacesList, err := namespaces.List(k8sapi.ListOptions{LabelSelector: labels.Everything()})
-		if err != nil {
-			htp.Error(w, http.StatusInternalServerError, "cannot get namespaces (%s)", err)
-			return
-		}
-		// TODO: delete concurrently https://github.com/deis/k8s-claimer/issues/49
-		for _, namespace := range namespacesList.Items {
-			if namespace.Name != "kube-system" && namespace.Name != "default" {
-				namespaces.Delete(namespace.Name)
-			}
-		}
+		// namespacesList, err := namespaces.List(k8sapi.ListOptions{LabelSelector: labels.Everything()})
+		// if err != nil {
+		// 	htp.Error(w, http.StatusInternalServerError, "cannot get namespaces (%s)", err)
+		// 	return
+		// }
+		// // TODO: delete concurrently https://github.com/deis/k8s-claimer/issues/49
+		// for _, namespace := range namespacesList.Items {
+		// 	if namespace.Name != "kube-system" && namespace.Name != "default" {
+		// 		namespaces.Delete(namespace.Name)
+		// 	}
+		// }
```

Comment:
Ive already stepped away from my laptop. I can do that first thing tomorrow
On May 23, 2016 5:20 PM, "Aaron Schlesinger" <notifications@github.com>
wrote:

> In handlers/delete_lease.go
> <https://github.com/deis/k8s-claimer/pull/70#discussion_r64306002>:
>
> > -		for _, namespace := range namespacesList.Items {
> > -			if namespace.Name != "kube-system" && namespace.Name != "default" {
> > -				namespaces.Delete(namespace.Name)
> > -			}
> > -		}
> > +		// namespacesList, err := namespaces.List(k8sapi.ListOptions{LabelSelector: labels.Everything()})
> > +		// if err != nil {
> > +		// 	htp.Error(w, http.StatusInternalServerError, "cannot get namespaces (%s)", err)
> > +		// 	return
> > +		// }
> > +		// // TODO: delete concurrently https://github.com/deis/k8s-claimer/issues/49
> > +		// for _, namespace := range namespacesList.Items {
> > +		// 	if namespace.Name != "kube-system" && namespace.Name != "default" {
> > +		// 		namespaces.Delete(namespace.Name)
> > +		// 	}
> > +		// }
>
> @jchauncey <https://github.com/jchauncey> if you want, you can just
> delete this code. There's a fix in progress in #68
> <https://github.com/deis/k8s-claimer/pull/68>, and the code will change
> significantly anyway
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/deis/k8s-claimer/pull/70/files/817b77704401fcf1065965fb7d04d8c39f76f45c#r64306002>
>